pep-0012.rst

```diff
@@ -475,68 +475,71 @@ Footnotes containing the URLs from external targets will be generated
 automatically at the end of the References section of the PEP, along
 with footnote references linking the reference text to the footnotes.
 
-Text of the form "PEP x" or "RFC x" (where "x" is a number) will be
-linked automatically to the appropriate URLs.
+To refer to PEPs or RFCs, always use the ``:pep:`` and ``:rfc:`` roles,
+never hard-coded URLs.
+For example:
+
+.. code-block:: rst
+
+    See :pep:`1` for more information on how to write a PEP,
+    and :pep:`the Hyperlink section of PEP 12 <12#hyperlinks>` for how to link.
+
+This renders as:
+
+    See :pep:`1` for more information on how to write a PEP,
+    and :pep:`the Hyperlink section of PEP 12 <12#hyperlinks>` for how to link.
+
+PEP numbers in the text are never padded, and there is a space (not a dash)
+between "PEP" or "RFC" and the number; the above roles will take care of
+that for you.
 
 
 Footnotes
 ---------
 
 Footnote references consist of a left square bracket, a number, a
-right square bracket, and a trailing underscore::
+right square bracket, and a trailing underscore
```

Comment:
End with a colon or full stop?
```suggestion
right square bracket, and a trailing underscore:
```

Reply:
Oops, deleted both `:`s instead of just one